### ISSUE TITLE
Battleship: Responsive design in extra credit

### ISSUE THREAD

**Issue by damon314159** [TheOdinProject/curriculum#27752]
### Checks

- [X] This is not a duplicate of an existing issue (please have a look through our [open issues list](https://github.com/TheOdinProject/curriculum/issues) to make sure)
- [X] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/.github/blob/main/CONTRIBUTING.md)
- [X] Would you like to work on this issue?

### Describe your suggestion

In the extra credit section, it states you can make a two player mode, with a pass over screen, but more importantly, "Make sure the game is playable on a mobile screen". However Battleships is before Advanced HTML/CSS where the concepts required to do this effectively are taught. This runs against much of the rhetoric of trusting the process and the order things are laid out in

### Path

Node / JS

### Lesson Url

https://www.theodinproject.com/lessons/node-path-javascript-battleship

**Comment by MaoShizhong** [comment]
Thought this was something somehow added in a recent-ish PR but looking through the lesson history, somehow it's been around pretty much forever.

100% agree that step is not appropriate to include, even in an extra credit section.

**Comment by MaoShizhong** [comment]
Just checking in @damon314159 and letting you know that since this issue has already been assigned to you, #27770 purposely doesn't address this part so if you were waiting on that, you're good to go whenever.

**Comment by damon314159** [comment]
Sorry I completely forgot this one. Will get it done asap

**Comment by MaoShizhong** [comment]
Legit thought you were waiting on Takinabradley's Battleship PR :shrug:
